1_fetch/src/find_oldest_sites.R

```diff
@@ -1,3 +1,11 @@
+#' Identify oldest USGS gage
+#'
+#' Identifies the oldest USGS gage within a user-specified state and
+#' for a user-specified parameter
+#'
+#' @param state chr, two letter state abbreviation (e.g., "WI")
+#' @param parameter chr, six-digit USGS parameter code
+#'
```

Comment:
💯 Adding in function documentation!

Reply:
It's to help out "future Julie". Pipelines 3 is a long one :D

Thanks for the review